weblate/templates/trans/alert/nomaskmatches.html

```diff
@@ -4,6 +4,6 @@
 <p>{% trans "There are no translations in this component." %}</p>
 
 <ul>
-  <li>{% blocktrans with filemask=component.filemask %}Check whether file mask <code>{{ filemask }}</code> matches files in your repository.{% endblocktrans %}</li>
-  <li>{% trans "You can also start new translations directly in Weblate, it will create files based on the file mask." %}</li>
+  <li>{% blocktrans with filemask=component.filemask %}Check whether the filemask <code>{{ filemask }}</code> matches files in your repository.{% endblocktrans %}</li>
```

Comment:
```suggestion
  <li>{% blocktrans with filemask=component.filemask %}Check whether the filemask <code>{{ filemask }}</code> matches files in your VCS repository.{% endblocktrans %}</li>
```

Reply:
It has shown in the past that many people don't understand what VCS stands for.

Reply:
Didn't know where to put the dash, but I think it is correct now.